web/src/lib/components/faces-page/search-people.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+  import SearchBar from '$lib/components/elements/search-bar.svelte';
+  import { maximumLengthSearchPeople, timeBeforeShowLoadingSpinner } from '$lib/constants';
+  import { handleError } from '$lib/utils/handle-error';
+  import { searchNameLocal } from '$lib/utils/person';
+  import { searchPerson, type PersonResponseDto } from '@immich/sdk';
+
+  export let searchName: string;
```

Comment:
Does this actually need to be exported?

Reply:
Yep, so the parent component know what's the user input, it allowed me to simplify [that part](https://github.com/immich-app/immich/pull/9082/files#diff-206e8cab8ac2af387ea424aad5b67c8669f615a66b8d83eb66271aa81d124998L444-R426) thanks to that : 
```ts
$: showPeople = searchName ? searchedPeopleLocal : people.filter((person) => !person.isHidden)
```

Reply:
Oh now it's starting to make sense to me....

Reply:
Yeah at the end, parent components "just" get a nice searchbar and a list of suggest people, they can do what they want with that list (most of the time filtering) and `searchName` is a nice way to know when the user is searching a person or not